src/components/Calculator.jsx

```diff
@@ -1,69 +1,63 @@
 /* eslint-disable linebreak-style */
 /* eslint-disable react/no-array-index-key */
 /* eslint-disable arrow-body-style */
-/* eslint-disable react/prefer-stateless-function */
 import './Calculator.css';
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import ButtonWraper from './ButtonWraper';
 import Screen from './Screen';
 import Button from './Button';
 import calculate from '../logic/calculate';

```

Comment:
[OPTIONAL] When using the modulo with 0 it crashes the app. I think this should work in the same manner as dividing by 0 does. This recommendation does encourage you to modify operate.js, which was given to you, so feel free not to follow it.

![image](https://user-images.githubusercontent.com/79114233/143487861-f34c186a-0ca7-4828-8f1f-def0c763da0c.png)